src/Options.js

```diff
@@ -247,13 +246,51 @@ export class Options {
     return mismatches;
   }
 
+  transformView() {
```

Comment:
Since we don't need to dynamically compute the views I just used a non-enumerable property initialized in the constructor.